examples/python/api_demo/main.py

```diff
@@ -277,18 +279,51 @@ def run_extension_component() -> None:
     )
 
 
+def run_image_tensors() -> None:
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    img_path = f"{dir_path}/rerun_logo_color_alpha.png"
```

Comment:
we have this logo already checked in here https://github.com/rerun-io/rerun/blob/main/crates/re_ui/data/logo_dark_mode.png
any reason not to reuse it?

Reply:
Missed it!